Monika After Story/game/script-topics.rpy

```diff
@@ -3483,6 +3483,56 @@ label monika_penname:
     m "There's no need to know more about me though, [player]."
     m "You already know that I'm in love with you after all~"
 
+init 5 python:
+    for key in ['change name']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_changename')
+
+label monika_changename:
+    if persistent.said_no == True:
+        m "Have you changed your mind, [player]?"
+        m "That's good to hear."
+    m "You want to change your name?"
+    menu:
+        "Yes":
+            m "Just type 'Nevermind' if you change your mind."
+            $ done = False
+            while not done:
+                $ tempname = renpy.input("What do you want me to call you?")
```

Comment:
Refer to what I said in `script-story-events.rpy`

Reply:
Null